build.psm1

```diff
@@ -1578,6 +1578,8 @@ function Start-PSxUnit {
 
```

Comment:
@KirkMunro, your last commit had 21 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

TestDebugJobIPC
```powershell
FAIL The term 'Start-ThreadJob' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.:TestInvokeError
Expected: True
Actual:   False
   at PSTests.Sequential.LocalDebuggerTests.TestDebugJobIPC() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 229
```
TestDebugJobWinRM
```powershell
FAIL A parameter cannot be found that matches parameter name 'BreakAll'.:TestInvokeError
Expected: True
Actual:   False
   at PSTests.Sequential.LocalDebuggerTests.TestDebugJobWinRM() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 280
```
TestRemoteDebuggerBreak
```powershell
System.Management.Automation.Remoting.PSRemotingTransportException : Connecting to remote server localhost failed with the following error message : The client cannot connect to the destination specified in the request. Verify that the service on the destination is running and is accepting requests. Consult the logs and documentation for the WS-Management service running on the destination, most commonly IIS or WinRM. If the destination is the WinRM service, run the following command on the destination to analyze and configure the WinRM service: "winrm quickconfig". For more information, see the about_Remote_Troubleshooting Help topic.
   at System.Management.Automation.Runspaces.AsyncResult.EndInvoke() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\AsyncResult.cs:line 233
   at System.Management.Automation.Runspaces.Internal.RunspacePoolInternal.EndOpen(IAsyncResult asyncResult) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\RunspacePoolInternal.cs:line 636
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.Open() in D:\a\1\s\src\System.Management.Automation\engine\remoting\client\RemoteRunspacePoolInternal.cs:line 899
   at System.Management.Automation.Runspaces.RunspacePool.Open() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\RunspacePool.cs:line 1105
   at System.Management.Automation.RemoteRunspace.Open() in D:\a\1\s\src\System.Management.Automation\engine\remoting\client\remoterunspace.cs:line 558
   at PSTests.Sequential.RemoteDebuggerTests.InitializeRunspace() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 346
   at PSTests.Sequential.DebuggerTestsBase.InitializeRunspace(EventHandler`1 debuggerStopHandler) in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 57
   at PSTests.Sequential.RemoteDebuggerTests.TestRemoteDebuggerBreak() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 912
```
TestRemoteDebuggerProcessCommand
```powershell
System.Management.Automation.Remoting.PSRemotingTransportException : Connecting to remote server localhost failed with the following error message : The client cannot connect to the destination specified in the request. Verify that the service on the destination is running and is accepting requests. Consult the logs and documentation for the WS-Management service running on the destination, most commonly IIS or WinRM. If the destination is the WinRM service, run the following command on the destination to analyze and configure the WinRM service: "winrm quickconfig". For more information, see the about_Remote_Troubleshooting Help topic.
   at System.Management.Automation.Runspaces.AsyncResult.EndInvoke() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\AsyncResult.cs:line 233
   at System.Management.Automation.Runspaces.Internal.RunspacePoolInternal.EndOpen(IAsyncResult asyncResult) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\RunspacePoolInternal.cs:line 636
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.Open() in D:\a\1\s\src\System.Management.Automation\engine\remoting\client\RemoteRunspacePoolInternal.cs:line 899
   at System.Management.Automation.Runspaces.RunspacePool.Open() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\RunspacePool.cs:line 1105
   at System.Management.Automation.RemoteRunspace.Open() in D:\a\1\s\src\System.Management.Automation\engine\remoting\client\remoterunspace.cs:line 558
   at PSTests.Sequential.RemoteDebuggerTests.InitializeRunspace() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 346
   at PSTests.Sequential.DebuggerTestsBase.InitializeRunspace(EventHandler`1 debuggerStopHandler) in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 57
   at PSTests.Sequential.RemoteDebuggerTests.TestRemoteDebuggerProcessCommand() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 1194
```
TestRemoteDebuggerCommandBreakpointUpdated
```powershell
System.Management.Automation.Remoting.PSRemotingTransportException : Connecting to remote server localhost failed with the following error message : The client cannot connect to the destination specified in the request. Verify that the service on the destination is running and is accepting requests. Consult the logs and documentation for the WS-Management service running on the destination, most commonly IIS or WinRM. If the destination is the WinRM service, run the following command on the destination to analyze and configure the WinRM service: "winrm quickconfig". For more information, see the about_Remote_Troubleshooting Help topic.
   at System.Management.Automation.Runspaces.AsyncResult.EndInvoke() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\AsyncResult.cs:line 233
   at System.Management.Automation.Runspaces.Internal.RunspacePoolInternal.EndOpen(IAsyncResult asyncResult) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\RunspacePoolInternal.cs:line 636
   at System.Management.Automation.Runspaces.Internal.RemoteRunspacePoolInternal.Open() in D:\a\1\s\src\System.Management.Automation\engine\remoting\client\RemoteRunspacePoolInternal.cs:line 899
   at System.Management.Automation.Runspaces.RunspacePool.Open() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\RunspacePool.cs:line 1105
   at System.Management.Automation.RemoteRunspace.Open() in D:\a\1\s\src\System.Management.Automation\engine\remoting\client\remoterunspace.cs:line 558
   at PSTests.Sequential.RemoteDebuggerTests.InitializeRunspace() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 346
   at PSTests.Sequential.RemoteDebuggerTests.InitializeRunspace(EventHandler`1 breakpointUpdatedHandler) in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 353
   at PSTests.Sequential.RemoteDebuggerTests.TestRemoteDebuggerCommandBreakpointUpdated() in D:\a\1\s\test\xUnit\csharp\test_Debugger.cs:line 556
```